examples/remarketing/upload_call_conversion.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Imports offline call conversion values for calls related to your ads.
+
+To set up a conversion action, run the add_conversion_action.py example.
+"""
+
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+def main(
+        client,
+        customer_id,
+        conversion_action_id,
+        caller_id,
+        call_start_date_time,
+        conversion_date_time,
+        conversion_value,
+    ):
+    """Imports offline call conversion values for calls related to your ads.
+
+    Args:
+        client: An initialized GoogleAdsClient instance.
+        customer_id: The client customer ID string.
+        conversion_action_id: The ID of the conversion action to upload to.
+        caller_id: The caller ID from which this call was placed. Caller ID is
+            expected to be in E.164 format with preceding '+' sign,
+            e.g. '+16502531234'.
+        call_start_date_time: The date and time at which the call occurred. The
+            format is 'yyyy-mm-dd hh:mm:ss+|-hh:mm',
+            e.g. '2019-01-01 12:32:45-08:00'.
+        conversion_date_time: The the date and time of the conversion (should be
+            after the click time). The format is 'yyyy-mm-dd hh:mm:ss+|-hh:mm',
+            e.g. '2019-01-01 12:32:45-08:00'.
+        conversion_value: The conversion value.
+    """
+    # Get the ConversionUploadService client.
+    conversion_upload_service = client.get_service(
+        "ConversionUploadService", version="v5"
+    )
+
+    # Create a call conversion in USD currency.
+    call_conversion = client.get_type("CallConversion", version="v5")
+    call_conversion.conversion_action = client.get_service(
+        "ConversionActionService", version="v5"
+    ).conversion_action_path(customer_id, conversion_action_id)
+    call_conversion.caller_id = caller_id
+    call_conversion.call_start_date_time = call_start_date_time
+    call_conversion.conversion_date_time = conversion_date_time
+    call_conversion.conversion_value = float(conversion_value)
```

Comment:
if you filter at the argparse level, you should remove the cast

Reply:
Done.